scripts/pre_push_hook.py

```diff
@@ -59,6 +59,10 @@
 
 
 class ChangedBranch(object):
+    """Class containing methods for changing branch in which modified file is
```

Comment:
Our conversation on this docstring has gone as follows:

> [seanlip]
> I think you should explain that this is a context manager class (please look up what that is). Normally a class doesn't "do" things so I was confused when I first read the docstring.
>
> [keshav]
> done
>
> [seanlip]
> I don't think you addressed this. Did you look up what a context manager class is?
>
> At the very least, the docstring should explain that this is a context manager class, as I suggested in my previous review comment.

This is still not resolved. I don't see a reply to the question in my previous comment and I don't see any mention of "context manager class" in the docstring.

Reply:
sorry sir , i will resolve it soon.
thanks

Reply:
@import-keshav, this is still unresolved. Please reply here after looking at this context and make relevant changes.

Reply:
i understand what context manager class is , i will resolve it thanks.